Amazon.IonObjectMapper.Test/TestCustomSerializersFactory.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Collections.Generic;
+using Amazon.IonDotnet;
+
+namespace Amazon.IonObjectMapper.Test {
+public class CourseSerializerFactory : IonSerializerFactory<Course>
```

Comment:
Nit: Should be indented one in from the namespace?

Reply:
Fixed.